.evergreen/spec.patch

```diff
@@ -1,15 +1,19 @@
---- mongo-cxx-driver.spec.orig	2020-08-17 16:22:55.490400655 -0400
-+++ mongo-cxx-driver.spec	2020-08-17 16:26:24.490702714 -0400
-@@ -1,6 +1,8 @@
+--- mongo-cxx-driver.spec.orig	2021-07-21 13:05:46.139038889 -0400
++++ mongo-cxx-driver.spec	2021-07-22 18:23:45.099314489 -0400
+@@ -1,17 +1,17 @@
  # for better compatibility with SCL spec file
  %global pkg_name mongo-cxx-driver
  
 +%define _unpackaged_files_terminate_build 0
 +
  Name:           mongo-cxx-driver
- Version:        3.4.1
- Release:        4%{?dist}
-@@ -10,8 +12,6 @@
+-Version:        3.4.1
+-Release:        5%{?dist}
++Version:        3.6.1
```

Comment:
Should this be our latest tagged release 3.6.4?

Reply:
This does appear to be introduced in this change. Was this pulled from the latest spec file? Comparing with https://src.fedoraproject.org/rpms/mongo-cxx-driver/blob/rawhide/f/mongo-cxx-driver.spec it appears to still be 3.4.1.

Reply:
Based on our pattern with the C driver, this should really be 3.7.0.